src/charm.py

```diff
@@ -139,6 +139,11 @@ def cluster_initialized(self):
         """Returns True if the cluster is initialized."""
         return self.peers.data[self.app].get("units-added-to-cluster", "0") >= "1"
 
+    @property
+    def unit_initialized(self):
```

Comment:
I wondering if this is a better approach for the `CONFIGURED_FILE` thing, since can serve the same purpose - but let that be to a soon to happen housekeeping